Application/src/view/visuals/component/Message.java

```diff
@@ -20,31 +20,25 @@
     private int offset; // offset for the message "ordering".
     private int class_size; // identifier for the current class size so that messages can scale.
     private int animationBounds = 0; //Starts and stops the animation at the correct bounds.
-    private boolean keepAnimating = true; // State of the animation. Beginning or ending.
+    private boolean animating = true; // State of the animation. Beginning or ending.
     private boolean switchImage; // Keeps track of which image to show.
     private double messageScale = 1.5;
     private int selfCallCounter = 1; // Used for the hardcoded trajectory of self referencing message.
-    private boolean switchDirection = false; // Keeps track whether a the dragon's flying direction should be switched
```

Comment:
how's the flying direction handled now?

Reply:
I removed a redundant variable,
directionSwitched is used now.

Reply:
+1